### ISSUE TITLE
Services.yaml configuration

### ISSUE THREAD

**Issue by garbast** [TYPO3GmbH/blog#157]
Please provide a Services.yaml configuration. This would make autowiring in extending context possible.
Example for minimum file.
```
services:
  _defaults:
    autowire: true
    autoconfigure: true
    public: false

  T3G\AgencyPack\Blog\:
    resource: '../Classes/*'
```

**Comment by helsner** [comment]
could you provide a PR for this, please?

**Comment by Lefaux** [comment]
Im 99.9% sure that this is fixed by #162 
If not, ping me again and I'll reopen the topic

**Comment by garbast** [comment]
You are right. My issue was from 1 August the other was not present then. So now the functionality is in already. Thank you.
